cmd/csi-snapshotter/main.go

```diff
@@ -90,6 +93,7 @@ var (
 
```

Comment:
Please remove this flag enableVolumeGroupSnapshots and add a release note that this is replaced by the feature gate.